src/en/news/blog/2022/scaletesting-with-pawsey/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Quincy @ Scale: Testing with the Pawsey Supercomputing Centre"
+date: "2022-07-11"
+author: "Paul Cuzner"
+tags:
+  - scalability
+  - quincy
+  - cephadm
+---
+
+During the development of new features, one of the most difficult considerations for Ceph developers is whether a new feature needs to be validated at scale. Our labs allow us to test new features in clusters of up to X,000 OSDs, and in most circumstances this works well!
```

Comment:
are we leaving the X,000 or is that to be filled in later?

Reply:
@adk3798 I raised the question about internal scale testing limits on the quincy@scale channel - but no reply. Do you know what the upper limit to our upstream testing is?